addon/services/rollbar.js

```diff
@@ -10,11 +10,17 @@ export default Ember.Service.extend({
   }),
 
   config: Ember.computed(function() {
-    return Ember.getOwner(this).resolveRegistration('config:environment').emberRollbarClient;
+    const applicationConfig = Ember.getOwner(this).resolveRegistration('config:environment');
+    const code_version = applicationConfig.APP.version;
```

Comment:
Basically, we can't use app versions as code version :/ We need to have a git hash here (or app version + hash)
app version can be the same in many commits :/

Reply:
Yes, but I saw, that you are already handling that in a draft of another PR that can be an improvement for this one.

Reply:
No one knows when it will be merged :/ Still waiting for some help there...